eq-author/src/App/section/Design/withDeleteSection.js

```diff
@@ -32,6 +32,7 @@ export const handleDeletion = (
   const questionnaire = data.deleteSection;
   const { sectionId, questionnaireId } = params;
 
+  // this will need to be modified similar to `withDeletePage`
```

Comment:
```suggestion
```

Or, does this still need to be done?

Reply:
It still needs to be done. The back-end part is done but will need that in there to remember to do the front-end. Also as a reminder to do it similarly to `withDeletePage`